src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+    public String getRandomBall() {
+        ColorSupplier colSup = new ColorSupplier();
+        Ball ball = new Ball();
+        ball.setColor(colSup.getRandomColor().toString());
+
+        Random index = new Random();
```

Comment:
Why do we use name `index` here?
index - is something relative to arrays and other data structures and maybe not to balls